### DISCUSSION TITLE
[Feature] Keyboard shortcut to toggle app preset volume level(s)

### DISCUSSION THREAD

**Discussion by Coldblackice** [File-New-Project/EarTrumpet#1650] [Ideas]
It would be great if there was an ability to preset multiple volume levels for a particular app, e.g. A) 100%, B) 10%, and then be able to set a keyboard shortcut to toggle/alternate between that app's volume levels.

There are some apps with volumes that need to frequently be lowered for periods of time, necessitating an opening and reopening of EarTrumpet's flyout to change an app's volume slider back and forth.

**Comment by riverar** [comment]
Try out EarTrumpet Actions in the dev builds (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller). Should work already!
